src/global/less/corporate-ui/forms.less

```diff
@@ -108,26 +108,27 @@ textarea[placeholder] {
 
     &.input-sm {
       background-size: 27px;
-      border-radius: 3px;
+      border-radius: @input-border-radius;
       }
 
       &.input-lg {
        background-size: 50px;
-       border-radius: 3px;
+       border-radius: @input-border-radius;
       }
 
     .app & {
       background-size: 27px;
       padding-right: 28px;
+      border-radius: @input-border-radius;
```

Comment:
Why do we need app border-radius if the default have it, Should that be inherit from default?